src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.scss';
+import {
+  AddToDoList,
+  setTasks,
+  showTasks,
+  GetToDoList,
+  updateLocalStorage,
+} from './modules/AddToDoItems.js';
+import { clearAllCompletedTasks } from './modules/Completed.js';
+
+const newTodolistInput = document.querySelector('#add-to-do > input');
+const addTaskButton = document.querySelector('#add-btn');
+const clearCompletedTasksButton = document.querySelector('#btn-Clear-completed-todos');
+
+newTodolistInput.addEventListener('keyup', (e) => {
+  if (e.key === 'Enter') {
+    AddToDoList(newTodolistInput);
+    showTasks();
+  }
+});
+
+addTaskButton.addEventListener('click', () => {
+  AddToDoList(newTodolistInput);
+  showTasks();
+});
```

Comment:
- [ ] You have done incredibly well to implement the add items functionality 👏 👍 . However, the `Enter` keyboard and `add-btn` is not responding as intended. Therefore, ensure that the tasks are added to the local storage. This will help you review other functions operation of the to-do list application ☺️.

Video demo 😎 

https://user-images.githubusercontent.com/99394374/222904045-93050b57-7158-4f83-8eff-b0efc8ccce29.mp4